lib/galaxy/config/schemas/config_schema.yml

```diff
@@ -1473,7 +1474,7 @@ mapping:
 
       citation_url:
         type: str
-        default: https://galaxyproject.org/citing-galaxy
+        default: https://academic.oup.com/nar/article/48/W1/W395/5849904
```

Comment:
I don't understand the reason for this change, the "Citing Galaxy" page have clear directions on which paper(s) to cite.

Reply:
This is what @nekrut requested. Any insight, Anton?

Reply:
We are trying to decrease the number of hub pages one needs to maintain. An alternative could be an API call to PubMed with PMIDs of publications we want

Reply:
I agree with lowering the hub pages to maintain. In this case, however, we need to maintain either the hub page or the Galaxy source. I prefer the hub page here, this gives us more freedom.

Reply:
@assuntad23 -> keep the hub page